Mathlib/SetTheory/Ordinal/Basic.lean

```diff
@@ -1145,6 +1145,9 @@ theorem gc_ord_card : GaloisConnection ord card := fun _ _ => ord_le
 theorem lt_ord {c o} : o < ord c ↔ o.card < c :=
   gc_ord_card.lt_iff_lt
 
+theorem pos_iff_ord_pos {c : Cardinal} : 0 < c ↔ 0 < c.ord := by
```

Comment:
This should be a simp lemma in the opposite direction, and it'd be nice to add `=` and `≤` versions of it, as well as those for `Ordinal.card`.

Reply:
Not sure the `≤` is that interesting 😁 

But yes please have `ord_pos`, `ord_eq_zero`, `ord_ne_zero`

Reply:
> Not sure the `≤` is that interesting 😁
> 
> But yes please have `ord_pos`, `ord_eq_zero`, `ord_ne_zero`

Most of these already exist under different names (the `Ordinal.card` versions as well). The `card` positivity one is `Ordinal.zero_lt_card`. Should this be renamed or should the `ord` one be named similarly instead of `ord_pos`?

Reply:
I think that should be renamed to `Ordinal.card_pos` too.

By the way, `ord_ne_zero` isn't actually needed, since `simp` simplifies `a ≠ b` to `¬ a = b` allowing it to use the `eq` theorems on `ne` statements. You can still add it if you think it's convenient for manual rewriting, though.

Reply:
Yea I just won't make it a simp lemma.